src/types.ts

```diff
@@ -63,28 +64,25 @@ export type Invoice = {
     faelle: Abrechnungsfall[];
 };
 
-export type Leistungserbringer = {
+export type Institution = {
```

Comment:
Should the type `Institution` in `src/kostentraeger/types` be renamed?

Reply:
I think it's fine. We probably won't need to expose both and in case we have to, we can still expose one under a different name.